File: RemoveDuplicates/removedupl.py
from typing import List

def removeDupl(elements: List) -> List:
    sol = []
    for elem in elements:
        if elem in sol:
            continue
        sol.append(elem)
    
    return sol

def main():
    test = [1,2,2,3]
    print(removeDupl(test))

if __name__ == "__main__":
    main()

File: RemoveDuplicates/removedupl.java
import java.util.*;

public class removedupl {
   
    public static ArrayList<Integer> removeduplicates(ArrayList<Integer> elements){
        ArrayList<Integer> solution = new ArrayList<>();

        for (int ele : elements){
            boolean dupl = false;
            for(int sol : solution){
                if(ele == sol){
                    dupl = true;
                    continue;
                }
            }
            if(dupl == false){
                solution.add(ele);
            }
        }
        return solution;
    }

    public static void main(String[] args){
        ArrayList<Integer> test = new ArrayList<>();
        test.add(1);
        test.add(2);   
        test.add(2);   
        test.add(3);                                                                                                                                                                                                                                                                                               
        System.out.println(removeduplicates(test));
    }
}

